src/platform/brand-consolidation/va-subdomain.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Returns whether the current hostname is a VA subdomain
+ * @returns {boolean}
+ * @module platform/brand-consolidation/is-subdomain
+ */
+export default function isVASubdomain() {
+  return !!window.location.hostname.match(
+    /(cem|benefits|telehealth|volunteer|ptsd|nutrition).va/,
+    'g',
```

Comment:
Two thoughts on this regex: first, should we make it case insensitive, just to be extra safe? Second, maybe it should work in the opposite way; that is, check to see if they are *not* on a subdomain, so that we don't have to maintain a list of valid subdomains to check against.

Reply:
We never got a definitive list of subdomains that should have it and no one seems to know how many sites use this Team Site template.

Reply:
Okay, so there *is* a list of subdomains that we want to match against; it's not just a matter of checking if the host is *a* VA subdomain. In that case I'd suggest renaming this to something a little clearer, like `isVATeamSiteDomain()`.

Reply:
Good idea, updated.